src/base/bittorrent/bencoderesumedatastorage.cpp

```diff
@@ -290,6 +290,8 @@ BitTorrent::LoadResumeDataResult BitTorrent::BencodeResumeDataStorage::loadTorre
     lt::add_torrent_params &p = torrentParams.ltAddTorrentParams;
 
     p = lt::read_resume_data(resumeDataRoot, ec);
+    if (ec)
+        return nonstd::make_unexpected(tr("Cannot parse resume data: %1").arg(QString::fromStdString(ec.message())));
```

Comment:
Other strings are formatted like this:
```suggestion
        return nonstd::make_unexpected(tr("Cannot parse resume data. Error: %1").arg(QString::fromStdString(ec.message())));
```

Reply:
> Other strings are formatted like this:

All the strings within the same scope this one appears in are formatted in the same way.
The string you're referring to is a log message. However, this one is the error description itself, which in the log message comes after the `Error:` (or `Reason:`) preamble.
Anyway, it uses the same format as all the other similar strings. So if someone doesn't agree with it, that's a separate issue.